### PR TITLE
Graphclust update

### PR DESCRIPTION
**Pull request by mmiladi**
This PR adds feature to report clustering quality metrics if input data is labeled.

### CODE REVIEW

File: tools/GraphClust/CollectResults/evaluation.py [hunk 1]
```diff
@@ -46,3 +47,31 @@ def sh(script):
     toWrite += listOfClasses[i] + "\t" + listOfClusters[i] + '\n'
 with open("RESULTS/fullTab.tabular", "w") as full:
     full.write(toWrite)
+
+
+listOfClasses = []
+listOfClusters = []
+pattern = re.compile("^RF.*$")
+
+with open("RESULTS/fullTab.tabular", 'r') as f:
```
Comment (bgruening) [inline tools/GraphClust/CollectResults/evaluation.py]: Why are you opening the same file twice? Could you not put the if condition into with and check the first line with your regex?

Comment (mmiladi) [inline tools/GraphClust/CollectResults/evaluation.py]: @eteriSokhoyan It seems one can check the pattern on first element of listOfClasses and not open the file second time. Do yo agree?

Comment (eteriSokhoyan) [inline tools/GraphClust/CollectResults/evaluation.py]: Yes sure, it's possible

Comment (mmiladi) [inline tools/GraphClust/CollectResults/evaluation.py]: @bgruening @eteriSokhoyan  Below commit should fix this.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**